deployment/localsettings.py

```diff
@@ -3,11 +3,11 @@
         'ENGINE': 'django.db.backends.postgresql',   # postgresql', 'mysql', 'sqlite3' or 'oracle'.
         'NAME': 'evap',                         # Or path to database file if using sqlite3.
         'USER': 'evap',                         # Not used with sqlite3.
-        'PASSWORD': 'evap',             # Not used with sqlite3.
+        'PASSWORD': 'evap',                     # Not used with sqlite3.
         'HOST': '127.0.0.1',                    # Set to empty string for localhost. Not used with sqlite3.
         'PORT': '',                             # Set to empty string for default. Not used with sqlite3.
         'CONN_MAX_AGE': 600,
     }
 }
 
-SECRET_KEY = "<%= @secret_key -%>"
+SECRET_KEY = "xvSFTFgwc8v5oYdB2F6oRVCShbZNAC"
```

Comment:
What do you think about leaving some kind of placeholder here and replacing it from the provision_vagrant_vm.sh script? We are already using sed in that file. If we leave this line like `SECRET_KEY = "${SECRET_KEY}"`, a call to sed like `sed -n -e "s/\${SECRET_KEY}/$(sudo head /dev/urandom | tr -dc A-Za-z0-9 | head -c 32)" localsettings.py` could do the job.

Reply:
yeah, i'll use that, thanks. admittedly i was simply too lazy to create that oneliner and i didn't care enough.